tide_core.install

```diff
@@ -587,3 +587,49 @@ function tide_core_update_8021() {
   $config_storage = \Drupal::service('config.storage');
   $config_storage->write($view, $source->read($view));
 }
+
+/**
+ * Enable scheduled transitions for each content type.
+ */
+function tide_core_update_8022() {
```

Comment:
I think this logic is better placed in the modules that declare the content types. Alternatively, so that we're not being too prescriptive, it may be better placed in each project.

Reply:
Cool, I am going to move this hook to each content-type module.

Reply:
I can be convinced either way. Scheduled transitions should be across the board is my first thought. Will it feel inconsistent if some content types offer it and others don't?

Reply:
@anthony-malkoun isn't that all the content-type should offer this if schedule transition is enabled?